src/components/top-bar/top-bar.stories.tsx

```diff
@@ -122,15 +123,16 @@ export const ReactUsage: StoryObj<typeof TopBar> = {
   render: () => {
     return (
       <TopBar>
+        <TopBar.Logo href="/">
+          <ReapitLogo />
+        </TopBar.Logo>
         <TopBar.MainNav>
           <NavItem href="/">Button 1</NavItem>
           <NavItem href="/">Button 2</NavItem>
-          <NavItem href="/">Button 3</NavItem>
-          <NavItem href="/">Button 4</NavItem>
           <CSSContainerQuery condition={'(width < 1004px)'}>
+            <NavItem href="/">Button 3</NavItem>
+            <NavItem href="/">Button 4</NavItem>
             <NavItem href="/">Button 5</NavItem>
-            <NavItem href="/">Button 6</NavItem>
-            <NavItem href="/">Button 7</NavItem>
           </CSSContainerQuery>
           <CSSContainerQuery condition={'not (width < 1005px)'}>
             <Menu>
```

Comment:
same as above, need to reduce the amount of button here, so it looks good in the mdx docs, since mdx docs display the preview in smaller container, while stories has full page.